app/Event/EventJSONLDServiceProvider.php

```diff
@@ -172,7 +174,9 @@ function () use ($container): EventLDProjector {
                     new EventTypeResolver(),
                     $container->get('config')['base_price_translations'],
                     new VideoNormalizer($container->get('config')['media']['video_default_copyright']),
-                    $container->get('config')['completeness']['event']
+                    new CompletenessFromWeights(
+                        Weights::fromConfig($container->get('config')['completeness']['event'])
```

Comment:
I'm more in favor of failing fast because default values could lead to wrong completeness scores which we only notice when they are already calculated. A missing config is also an issue that needs to be addressed as fast as possible.